amr/run.wdl

```diff
@@ -1,21 +1,40 @@
 version 1.1
 
-workflow AMR {
+import "../short-read-mngs/host_filter.wdl" as host_filter
+
+workflow amr {
     input {
-        Array[File] non_host_reads
+        Array[File]? non_host_reads
+        File? raw_reads_0
+        File? raw_reads_1
         File contigs
         String docker_image_id
+        String host_filtering_docker_image_id = "czid-short-read-mngs" # default local value
         File card_json = "s3://czid-public-references/test/AMRv2/card.json"
         File kmer_db = "s3://czid-public-references/test/AMRv2/61_kmer_db.json"
         File amr_kmer_db = "s3://czid-public-references/test/AMRv2/all_amr_61mers.txt"
         File wildcard_data = "s3://czid-public-references/test/AMRv2/wildcard_data.tar.bz2"
         # Dummy values - required by SFN interface
         String s3_wd_uri = ""
     }
-
+    if (defined(raw_reads_0)) { 
+        call host_filter.czid_host_filter as host_filter_stage { 
+            input:
+            fastqs_0 = select_first([raw_reads_0]),
+            fastqs_1 = if defined(raw_reads_1) then select_first([raw_reads_1]) else None,
+            docker_image_id = host_filtering_docker_image_id,
```

Comment:
In this case we are using different docker images for host filtering compared to the AMR pipeline. If this causes a problem we can merge these into a large image